tests/test_basic_operation.py

```diff
@@ -42,7 +45,10 @@ def test_004_init_secondary():
     global node2
     node2 = cluster.create_datanode("/tmp/basic/node2")
     node2.create()
```

Comment:
Something like this would be good to test too, to see that the `--name` argument of create works:

```suggestion
    node2.create()
    node2.create(name="create_b")
    eq_(node2.get_nodename(), "create_b")
```

Reply:
Right, added now.